src/DynamoPackages/Properties/Resources.cs-CZ.resx

```diff
@@ -123,6 +123,9 @@
   <data name="CannotRemovePackageAssemblyTitle" xml:space="preserve">
     <value>Sestavu není možné aktualizovat</value>
   </data>
+  <data name="DownloadPackageDisabled" xml:space="preserve">
```

Comment:
These other locale resources will be added later with the localization job, right? Any reason to add them as empty value here?

Reply:
I'm not sure what the process is here. @QilongTang, can you clarify?